app/editor/src/hooks/api-editor/editor/useApiActions.ts

```diff
@@ -18,7 +18,7 @@ export const useApiActions = (
 
   return {
     getActions: (etag: string | undefined = undefined) => {
-      const config = !!etag ? { headers: { 'If-None-Match': etag } } : undefined;
+      const config = { headers: { 'If-None-Match': etag ?? '' } };
```

Comment:
Not certain if this was required, but it looked like the browser was adding the value if I didn't set it to blank.  Which can cause issues if the local storage doesn't have a copy of the list.